File: permission/src/main/java/com/jiangyang/permission/PermissionCallback.java
package com.jiangyang.permission;

import java.util.List;

/**
 * <pre>
 * @author : No.1
 * @time : 2018/5/2.
 * @desciption :
 * @version :
 * </pre>
 */

public interface PermissionCallback {
    /**
     * 获取权限成功回调
     */
    void onGranted();

    /**
     * 未授权权限
     *
     * @param list 未授权权限集合
     */
    void onDenied(List<String> list);
}

File: permission_kt/src/main/java/com/banzhi/permission_kt/PermissionCallback.kt
package com.banzhi.permission_kt

/**
 *<pre>
 * @author : No.1
 * @time : 2019/8/7.
 * @desciption :
 * @version :
 *</pre>
 */
interface PermissionCallback {
    /**
     * 获取权限成功回调
     */
    abstract fun onGranted()

    /**
     * 未授权权限
     *
     * @param list 未授权权限集合
     */
    abstract fun onDenied(list: List<String>)
}